gateway/reverse_proxy.go

```diff
@@ -538,21 +538,39 @@ func (p *ReverseProxy) ServeHTTPForCache(rw http.ResponseWriter, req *http.Reque
 	return resp
 }
 
+// This returns the expected 30 second default timeout in case
+// the ProxyDefaultTimeout value is undefined.
+func proxyDefaultTimeout(spec *APISpec) float64 {
```

Comment:
can we apply rules of GoDoc, starting func name `proxyDefaultTimeout returns...`